controller/feed-controller.js

```diff
@@ -0,0 +1,51 @@
+const db = require("../db/db");
+const {
+  STATUSCODE,
+  STATUS,
+  successResponse,
+  errorResponse,
+} = require("../utilities/response-utility");
+
+// Function to get all items (articles or gifs) from the database
+const getAllItems = async (tableName) => {
+  const query = `
+    SELECT *
+    FROM ${tableName}
+    ORDER BY created_on DESC;
+  `;
+  const result = await db.query(query);
+  return result.rows;
+};
+
+// Function to combine and sort items by date
+const combineAndSortItems = (items) => {
```

Comment:
```suggestion
const sortItems = (items) => {
```
This Is Only Sorting It